bn254/src/compression.rs

```diff
@@ -483,6 +403,9 @@ mod tests {
 
         test_cases.iter().for_each(|test| {
             let input = array_bytes::hex2bytes_unchecked(&test.input);
+            if input.len() < 192 {
+                return;
+            }
```

Comment:
Has the testing changed in any way by removing the input? I just wonder why you needed to check the length now.

Reply:
Yeah sorry for the confusion. I completely forgot about this myself and I should have mentioned it in the PR description. Before, there were two test cases, one for the `alt_bn128_pairing_test` in `lib.rs` and one for the `alt_bn128_compression_pairing_test_input` in `compression.rs`. They are actually the same set of tests except that the `alt_bn128_pairing_test` had an [empty](https://github.com/anza-xyz/solana-sdk/pull/216/files#diff-6ae2a2cde7e2bb582e7dd42bf93edffc1720aacf1225b0d75283333ec8f27770R45) case that does not make sense for compression. I ended up merging the two test cases into a single data file that include the empty case and then added a check to skip the empty case in the compression tests. I wrote a comment to clarify this.